evap/settings.py

```diff
@@ -352,6 +352,17 @@ class ManifestStaticFilesStorageWithJsReplacement(ManifestStaticFilesStorage):
 # Absolute filesystem path to the directory that will hold user-uploaded files.
 MEDIA_ROOT = os.path.join(BASE_DIR, "upload")
 
+### Evaluation progress rewards
+GLOBAL_EVALUATION_PROGRESS_REWARDS: list[tuple[float, str]] = [
+    (0, "0€"),
+    (0.25, "1.000€"),
+    (0.6, "3.000€"),
+    (0.7, "7.000€"),
+    (0.9, "10.000€"),
+]
+EXCLUDED_COURSE_TYPE_IDS: list[int] = []
+EXCLUDED_EVALUATION_IDS: list[int] = []
```

Comment:
On its own, this name does not convey the context. Maybe it would make sense to nest related settings at some point. For now, I'd like to add some kind of context prefix to these. I guess it's not a big problem if they get verbose, so maybe simply `GLOBAL_EVALUATION_PROGRESS_EXCUDED_COURSE_TYPE_IDS` and `GLOBAL_EVALUATION_PROGRESS_EXCUDED_EVALUATION_IDS`?